lib/mailtrap/client.rb

```
@@ -50,41 +32,96 @@ def send(mail)
+      attach_headers(request)
+      request
+    end
+
+    def attach_headers(request)
       request['Authorization'] = "Bearer #{api_key}"
       request['Content-Type'] = 'application/json'
       request['User-Agent'] = 'mailtrap-ruby (https://github.com/railsware/mailtrap-ruby)'
+    end
 
-      request
+    def perform_request(uri, request)
```

RuboCop (1.75.5)

[warning] Unused method argument - `uri`. If it's necessary, use `_` or `_uri` as an argument name to indicate that it won't be used. If it's unnecessary, remove it.

(Lint/UnusedMethodArgument)